--- src/base/bittorrent/session.cpp
+++ src/base/bittorrent/session.cpp
@@ -1220,8 +1220,10 @@ void Session::initMetrics()
     m_metricIndices.disk.numBlocksRead = lt::find_metric_idx("disk.num_blocks_read");
     Q_ASSERT(m_metricIndices.disk.numBlocksRead >= 0);

+#if (LIBTORRENT_VERSION_NUM < 20000)
     m_metricIndices.disk.numBlocksCacheHits = lt::find_metric_idx("disk.num_blocks_cache_hits");
     Q_ASSERT(m_metricIndices.disk.numBlocksCacheHits >= 0);
+#endif

     m_metricIndices.disk.writeJobs = lt::find_metric_idx("disk.num_write_ops");
     Q_ASSERT(m_metricIndices.disk.writeJobs >= 0);
